sdks/java/container/license_scripts/dep_urls_java.yaml

```
@@ -65,3 +65,7 @@ org.eclipse.jgit:
   '4.4.1.201607150455-r':
     license: "https://www.eclipse.org/org/documents/edl-v10.html"
     type: "Eclipse Distribution License - v1.0"
+zstd-jni:
+  '1.5.2-5':
+    license: "https://github.com/luben/zstd-jni/blob/master/LICENSE"
```

Should this be `https://raw.githubusercontent.com/luben/zstd-jni/master/LICENSE`?
```suggestion
    license: "https://raw.githubusercontent.com/luben/zstd-jni/master/LICENSE"
```

---

Yes, updated